recipes/tensorflow_io_gcs_filesystem/meta.yaml

```diff
@@ -19,6 +19,7 @@ requirements:
   build:
     - {{ compiler('cxx') }}
     - bazel 3.7.2
+    - tensorflow >=2.6.0,
```

Comment:
This hsould likely be a host dependency.

Reply:
And i really don't think you need the upper pin.

Reply:
Thanks for editing it :)